Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem "rake"
+gem "jeweler"
+gem "test-unit"
```

Comment:
I didn't set any versions since I wasn't sure what you'd developed it against.

Reply:
As above, no specific versions. Whatever was current in 2010 was fine :-)

Since `rake` had not even reached v1.0 back in 2010, and it's now at 12.1.0, I'm not sure what would be a sensible version to depend on. Maybe 10.0 would be reasonable, since it was apparently the start of a super-major version?

Jeweler moved onto its present major version of 2.x in 2014, which seems long enough ago that it's not unreasonable to expect people to have that, so perhaps depend on v2.0?

And by similar reasoning test-unit v3.0 looks reasonable to me.

Any thoughts?

Reply:
Since the `Gemfile` (as opposed to dependencies listed in the `.gemspec`) is really just for development dependencies we can specify recent versions.